src/chage.c

```diff
@@ -45,7 +45,7 @@
 /*
  * Global variables
  */
-const char *Prog;
+const char *Prog = "chage";
```

Comment:
To make sure we don't mess with it, you could use

```c
static const char  Prog[] = "chage";
```

Otherwise, the pointer can be changed.